cob_image_flip/ros/src/image_flip.cpp

```diff
@@ -615,9 +682,9 @@ void ImageFlip::disparityCallback(const stereo_msgs::DisparityImage::ConstPtr& d
 		for (int v = 0; v < disparity_image_turned.rows; v++)
 			for (int u = 0; u < disparity_image_turned.cols; u++)
 				disparity_image_turned.at<float>(v,u) = disparity_image.at<float>(disparity_image.rows-1-u,v);
-		rot_mat.at<double>(0,1) = -1.;
```

Comment:
Just leave it the old way in this case, because the rotation matrix is always the same format CV_64FC1.